Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/TabbedPage.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	using FormsElement = Forms.TabbedPage;
+
+	public static class TabbedPage
+	{
+		const string TranslucencyModePropertyName = "TranslucencyMode";
```

Comment:
creating a const that is used once is overkill

Reply:
I will update it, but I cannot agree that it's overkill. From my perspective, it's a good practice. Anyway, constants will be put in a literal pool.